webpack.config.js

```diff
@@ -0,0 +1,27 @@
+const path = require('path');
```

Comment:
Good heavens, webpack? You are unstoppable!

Reply:
Webpack just to uglify and make a testable solution. Webpack is easy as pie if you have no eternal modules, code splitting, css, html, or anything else.